src/main/kotlin/org/jitsi/nlj/transform/node/Node.kt

```diff
@@ -100,12 +100,22 @@ sealed class Node(
     }
 
     protected fun next(packetInfo: PacketInfo) {
+        if (PLUGINS_ENABLED) {
+            plugins.forEach { it.observe(this, packetInfo) }
+        }
+        // nextNode?.processPacket(packetInfo)
+        detailedNext(packetInfo)
+    }
+
+    protected fun nextFromChild(packetInfo: PacketInfo) {
```

Comment:
It seems like this will call plug-ins twice per node?

Reply:
Good catch.  Will fix that if we move forward.